cypress/integration/lsp-tokens/0-seed-account.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+
+// Fund 20 UMA to test account before running other tests.
+describe("Fund UMA to test wallet", () => {
```

Comment:
Are these expected to be run in the order 0 -> 1 -> 2, etc.

Or could 1, 2, 3, 4 be run as completely independent tests? If most are independent, would it make sense to move this out of a test and into the `before` block for each test? In general, independent test files (and test cases where possible) are great because they allow you to isolate them when debugging.

Reply:
I think test 0 has to be run first (to seed UMA to mint) -- it was easier to keep them separate in order to make it easier to develop.

Tests 1 and 2 (mint and settle) are totally independent, although test 2 does put time forward, so it would cause test 1 to fail. They are intended to be run in order due to moving blockchain time forward.

Reply:
Are there static timestamps encoded in the tests? Could we use the prevailing block time to make these antifragile with respect to moving time around?

I would argue for putting any setup in the `before()` block. If it's complicated, you could toss it into a utils file that gets imported into each test.